src/frontend/src/components/public/PIAFormTabs/intake/helper/messages.ts

```diff
@@ -76,7 +76,7 @@ export default {
       en: `Did you list personal information in question 3?`,
     },
     LinkText: {
-      en: `Personal information`,
+      en: `Personal information `,
```

Comment:
We can't use margins because of the way this is implemented. I've added the &nbsp; to the index file itself, as using this code cannot be done via injecting it from messages. Spacing is added this way all over the place, and in some cases is the only choice because it is added a prop to a component. We will need to create a solution to this if it is going to be a problem.